### DISCUSSION TITLE
Wyze Cam Battery Pro comparability?

### DISCUSSION THREAD

**Discussion by bhsuarez** [mrlt8/docker-wyze-bridge#1349] [General]
Is there any plan to get this working? I have two and would love to use this. Thanks

**Comment by xerootg** [comment]
Long story short, this isn't the project for that. Wyze used IotVideoSDK from tencent for your cameras, and it's kind of a technology mess. I do however have a project that is starting to mature for those cameras, [cryze_v2](https://github.com/xerootg/cryze_v2) but it's a bit... Involved compared to this project because the only libraries out there for these cameras are written for Android. I've got the whole stack running on Linux, in docker, but with compromises.

**Reply by bhsuarez** [reply]
Thanks. Looks interesting so I'd like to try it out. What version of Linux are you using?

**Reply by xerootg** [reply]
Thanks! I'm running arch, zen kernel, on the machine that hosts my non-development stuff (and the main branch of cryze_v2), and some very outdated Ubuntu install on my machine I do my development against.

**Comment by bhsuarez** [comment]
Now final question, would I be able to view these in Home Assistant?

--
Brian H. Suarez
***@***.***

**Reply by xerootg** [reply]
Yeah, add them as a generic camera in homeassistant. Rtsp, probably primarily over TCP is fine. Theoretically anything mediamtx supports, this should. Personally, I put everything into frigate and then use the HA/frigate integration, cause it's just more reliable that having HA itself handle streams. The Frigate/HA integrity uses a jpeg in cards, so it's just more resilient to network issues than having HA do either generic camera or ffmpeg based video streaming in my experience.

**Comment by almarazjm** [comment]
Hello, wondering if there are any updates on this topic?
